api/pom.xml

```diff
@@ -64,7 +64,11 @@
                 <groupId>biz.aQute.bnd</groupId>
                 <artifactId>bnd-maven-plugin</artifactId>
                 <version>2.4.1</version>
-
+                <configuration>
```

Comment:
The configuration should be in another place I have fixed it in https://github.com/eclipse/microprofile-opentracing/pull/203